README.md

```diff
@@ -9,10 +9,42 @@ This project uses CMake for setting up a build system, usually GNU Make. This is
  * Run CMake: ``cmake ..``. Note that the default build build type is *Release*. To build in debug mode, run CMake as follows: ``cmake -DCMAKE_BUILD_TYPE=Debug ..``.
  * To compile run: ``make``.
 
+### Pull down dependencies with Conan
```

Comment:
Do we really want to have all this stuff in the `README.md` or would it be better to move this to a wiki page. Conan is a rather special tool and one cannot expect users to have it installed. 
In particular installing conan with `pip install conan` globally on a system is rather dangerous. It can easily mess up your local Python installation.